workflow-examples/run_examples.sh

```diff
@@ -89,6 +92,24 @@ get_next_workflow() {
     -H 'Content-Type: application/json' | jq '.tasks[] | select(has("nextWorkFlow"))' | jq -r '.nextWorkFlow' | head -n 1
 }
 
+wait_project_start() {
+  echo_blue "******** Checking project is running ********"
+  for i in {1..1}
```

Comment:
`1..1` is a single iteration, right?

Reply:
meaning, `i` isn't progressing, so only one iteration and the condition on 108 will always be false?

Reply:
sorry, was testing it and I forget it.